x-pack/plugins/reporting/server/export_types/png_v2/png_v2.ts

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import apm from 'elastic-apm-node';
+import { LicenseType } from '@kbn/licensing-plugin/server';
+import { CancellationToken, TaskRunResult } from '@kbn/reporting-common';
+import { Writable } from 'stream';
+import {
+  finalize,
+  fromEventPattern,
+  lastValueFrom,
+  map,
+  mergeMap,
+  Observable,
+  of,
+  takeUntil,
+  tap,
+} from 'rxjs';
+import { SerializableRecord } from '@kbn/utility-types';
+import { LocatorParams } from '../../../common';
+import {
+  LICENSE_TYPE_CLOUD_STANDARD,
+  LICENSE_TYPE_ENTERPRISE,
+  LICENSE_TYPE_GOLD,
+  LICENSE_TYPE_PLATINUM,
+  LICENSE_TYPE_TRIAL,
+  PNG_JOB_TYPE_V2,
+  PNG_REPORT_TYPE_V2,
+  REPORTING_TRANSACTION_TYPE,
+} from '../../../common/constants';
+import { decryptJobHeaders, ExportType, generatePngObservable } from '../common';
+import { JobParamsPNGV2, TaskPayloadPNGV2 } from './types';
+import { getFullRedirectAppUrl } from '../common/v2/get_full_redirect_app_url';
+
+/*
+ * @TODO move to be within @kbn/reporting-export-types
+ */
+
```

Comment:
I suggest removing this, because I don't fully support the idea to move this code to a package. We can talk about it more, but I think it benefits the code overall to have this in the `reporting_export_types` plugin (moving dependencies out of reporting, make Reporting to be depended on, rather than import the export types into Reporting).
```suggestion
```

Reply:
😭